frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -504,14 +507,10 @@ function ReaderBookmark:onShowBookmark(match_table)
                 },
                 direction = BD.flipDirectionIfMirroredUILayout("east")
             }
-        }
-    }
-
-    self.bookmark_menu = CenterContainer:new{
-        dimen = Screen:getSize(),
-        covers_fullscreen = true, -- hint for UIManager:_repaint()
-        bm_menu,
+        },
+        show_parent = self.bookmark_menu,
```

Comment:
You then can remove the one at the bottom of the function:
`bm_menu.show_parent = self.bookmark_menu`

Reply:
Done